### DISCUSSION TITLE
Question in regards to archive(rsync) client conf file.

### DISCUSSION THREAD

**Discussion by fireboy0526** [ericaltendorf/plotman#594] [Q&A] [answered]
Hello all,

Going through the docs, I've learned how to setup the rsyncd.conf file for the destination server (where all my plots will be send to). However, I can't seem to find any configuration docs on how the plotter machines should setup their rsyncd.conf file. 

My plotter machine, which is running ubuntu 20.04.2 comes with rsync but it doesn't has a rsyncd.conf file within the /etc folder, which is where I'm currently loss.

Any help will be more then appreciated. 

Thank you.

**Comment by Purple-Tentacle** [comment, marked as answer]
rsyncd (d = daemon) is the server.
rsync is the client.
The plotters only use rsync to access the destination server. No need to run rsyncd on the plotters.

**Reply by fireboy0526** [reply]
I see. Thanks for clearing everything up for me!

However, if I do `sudo systemctl status rsync` does this show the status of the client or the server?

**Reply by altendky** [reply]
server
